src/software/cloud-computing/README.md

```diff
@@ -0,0 +1,54 @@
+# Cloud Computing
+
+Cloud computing is the delivery of various computing services, such as servers, storage, databases, and software, over the internet. It provides faster innovation, flexibility, and cost savings, as you only pay for the services you use, allowing you to lower your costs and scale based on your business needs.
+
+## Benefits of Cloud Computing
+
+- **Cost Efficiency**: Eliminate the upfront expense of hardware, software, and data centers by leveraging cloud infrastructure.
+- **Speed**: Quickly deploy services with just a few clicks, reducing time to market.
+- **Global Scale**: Scale services globally, delivering resources from the nearest location to ensure low latency and optimal performance.
+- **Productivity**: Focus on core business objectives rather than managing physical servers and infrastructure.
+- **Reliability**: Simplify data backup, disaster recovery, and business continuity through built-in redundancy.
+- **Security**: Use advanced security features such as encryption and multi-factor authentication (MFA) to protect sensitive data.
+- **Performance**: Improve performance by distributing workloads across multiple servers and geographic regions for speed and redundancy.
+
+## Types of Cloud Computing
+
+- **Public Cloud**: Services are owned and operated by third-party cloud providers and delivered over the internet. You pay for only for what you use(AKA Operating expense-OpEx).
+- **Private Cloud**: Infrastructure is owned and used exclusively by a single organization. They pay for all the Infrastructure upfront (AKA Capital expense-CapEx).
+- **Hybrid Cloud**: Combines public and private cloud environments.
+- **Multi Cloud**: Use multiple public cloud providers.
+
+</br>
+
+## Cloud Platform Models
+
+Cloud services come in more than one flavor. Choosing the right one will depend on your specific needs and how much fine control you'll need over the underlying gears and levers.
+
+**Infrastructure as a Service**  
+Infrastructure as a Service (IaaS) products generally simulate the look and feel you'd get from managing physical resources. laaS products give you direct access to a provider's compute, stor-age, and networking assets. Because it's you that's in there playing around at the hardware level, you—rather than the laaS provider-are responsible for the consequences of any bad configura-tions. The trade-off is that you get to closely configure every layer of your operating stack.
```

Comment:
make sure to remove the unneeded hyphens in this sentence

Reply:
Thanks for letting me know, I think it should be fixed now